packages/babel-plugin/README.md

```diff
@@ -0,0 +1,235 @@
+| <code>Types.LinkingObjects<T,&nbsp;N></code> | `linkingObjects`  |                 |                         | `T` is the type of objects, `N` is the property name of the relationship (as a string) |
+
+### Specifying schema properties as `static`s
+
+Additional schema properties can be specified by adding `static` properties to your class, as shown in the table below. See [the Realm documentation](https://www.mongodb.com/docs/realm/sdk/node/examples/define-a-realm-object-model/) for more details.
+
+| Static property | Type      | Notes                                                                      |
+| --------------- | --------- | -------------------------------------------------------------------------- |
+| `name`          | `string`  | Specifies the name of the Realm schema. Defaults to your class name.       |
+| `primaryKey`    | `string`  | Specifies the name of a property to be used as the primary key.            |
+| `embedded`      | `boolean` | Specifies this is an embedded schema.                                      |
+| `asymmetric`    | `boolean` | Specifies this schema should sync unidirectionally if using flexible sync. |
+
+For example:
+
+```ts
+import Realm from "realm";
+
+export class Task extends Realm.Object<Task, "description"> {
+  _id = new Realm.BSON.ObjectId();
+  description!: string;
+  isComplete = false;
+
+  static primaryKey = "_id";
+}
+```
+
+### Using decorators to index and remap properties
+
+The `@realm/babel-plugin` package exports decorators to allow you to specify certain properties should be indexed (using the `@index` decorators) or should remap to a Realm schema property with a different name (using the `@mapTo` decorator). To learn more about this functionality, see [the documentation](https://www.mongodb.com/docs/realm/sdk/react-native/examples/define-a-realm-object-model/#index-a-property).
+
+Note that use of decorators requires using the `@babel/plugin-proposal-decorators` plugin and for `experimentalDecorators` to be enabled in your `tsconfig.json`. There is currently no way to specifying properties to be indexed or remapped without using decorators.
+
+This table shows the available decorators:
+
+| Decorator | Parameters                    | Notes                                                                                                |
+| --------- | ----------------------------- | ---------------------------------------------------------------------------------------------------- |
+| `index`   | none                          | Specifies that the decorated property should be indexed by Realm.                                    |
+| `mapTo`   | `(realmPropertyName: string)` | Specifies that the decorated property should be stored as `realmPropertyName` in the Realm database. |
+
+The example below shows both decorators in use:
+
+```ts
+import Realm from "realm";
+import { mapTo, index } from "@realm/babel-plugin";
+
+export class Task extends Realm.Object {
```

Comment:
Do we need to change `Realm.Object` to `Realm.Object<Task>`?

Reply:
```suggestion
export class Task extends Realm.Object<Task> {
```